<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Garden Control Panel</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(138, 226, 52);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>431</width>
      <height>498</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QFormLayout" name="formLayout_8">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(193, 125, 17);</string>
         </property>
         <property name="text">
          <string>Sensor: 3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Moisture:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lbls3moist">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_16">
         <property name="styleSheet">
          <string notr="true">width: 100%;
height: 100%</string>
         </property>
         <property name="text">
          <string>Relay</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lbls3Relay">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Off</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="btns3Connect">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(78, 154, 6);</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="btns3Relay">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(78, 154, 6);</string>
         </property>
         <property name="text">
          <string>Relay</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout_9">
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Moisture:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lbls4moist">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Relay</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lbls4Relay">
         <property name="text">
          <string>Off</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="btns4Relay">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(78, 154, 6);</string>
         </property>
         <property name="text">
          <string>Relay</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="btns4Connect">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(78, 154, 6);</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(193, 125, 17);</string>
         </property>
         <property name="text">
          <string>Sensor: 4</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(193, 125, 17);</string>
         </property>
         <property name="text">
          <string>Sensor: 1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Moisture:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lbls1moist">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Relay</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lbls1Relay">
         <property name="text">
          <string>Off</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="btns1Relay">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(78, 154, 6);</string>
         </property>
         <property name="text">
          <string>Relay</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="btns1Connect">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(78, 154, 6);</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="btns3Relay_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(78, 154, 6);</string>
       </property>
       <property name="text">
        <string>Relay</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QFormLayout" name="formLayout_7">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(193, 125, 17);</string>
         </property>
         <property name="text">
          <string>Sensor: 2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Moisture:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lbls2moist">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Relay</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lbls2Relay">
         <property name="text">
          <string>Off</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="btns2Relay">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(78, 154, 6);</string>
         </property>
         <property name="text">
          <string>Relay</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="btns2Connect">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(78, 154, 6);</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QWidget" name="formWidget_5" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(252, 233, 79);</string>
       </property>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Temp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lblPumpTemp">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Humidity</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lblPumpHumidity">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Pressure</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="lblPumpPressure">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lblPumpStatus">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Relay</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="lblPumpRelay">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_18">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(196, 160, 0);</string>
          </property>
          <property name="text">
           <string>Pump Control</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="btns3Connect_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(78, 154, 6);</string>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
